src/components/Task/Drawer/Drawer.tsx

```diff
@@ -175,7 +183,11 @@ const TaskDrawer = ({
           </Box>
           <Box className={classes.content}>
             <AnimatePresence initial={false}>
-              <TabPanel key="1" value="1" className={classes.tabPanel}>
+              <TabPanel
+                key="1"
```

Comment:
What is `key`? I don't see it in the docs https://material-ui.com/api/tab-panel/#props. Is it React's `key`? This isn't a loop...

Reply:
Yeah not sure, maybe it was first implemented as a loop and then was just left in? I'll go ahead and remove it since it's not necessary.